### PR TITLE
PIN-540: smartcard redemption

### PR DESCRIPTION
**Pull request by Troi**

### CODE REVIEW

File: src/VoucherBundle/Controller/VendorController.php [hunk 1]
```diff
@@ -180,6 +185,140 @@ public function getSingleActionVendor(Vendor $vendor)
         return $this->getSingleAction($vendor);
     }
 
+    /**
+     * Get vendor purchase counts
+     *
+     * @Rest\Get("/vendors/{id}/purchases", name="get_all_vendor_purchases")
```
Comment (iarro) [inline src/VoucherBundle/Controller/VendorController.php:191]: Wrong enpoint name. You return only smartcard purchases, not ALL purchases. It should be defined in /smartcards

Comment (iarro) [reply src/VoucherBundle/Controller/VendorController.php:191]: All added functionality should be in SmartcardController

Comment (iarro) [reply src/VoucherBundle/Controller/VendorController.php:191]: Also, are there urls accesible by mobile app. In that case, it shoult be prefixed by vendor-app or offline-app

**Review by iarro — CHANGES_REQUESTED** [review]

File: src/VoucherBundle/Controller/VendorController.php (hunk 1 above)
Comment (Troi) [reply src/VoucherBundle/Controller/VendorController.php]: updated

mobile apps arent consumer of this endpoints

**Comment by Troi** [comment]
will be rebased

**Pull request closed without merging**